alembic/versions/33de75e55858_adding_datetime_column.py

```diff
@@ -0,0 +1,44 @@
+"""Adding datetime column
+
+Revision ID: 33de75e55858
+Revises: 525b125ec799
+Create Date: 2015-10-10 12:32:39.936107
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '33de75e55858'
+down_revision = '525b125ec799'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('campaign_anon_view', sa.Column('datetime', sa.DateTime(), nullable=False))
```

Comment:
If you can add `server_default` when making the column, pointing it to an existing column, that's cool too. Just remove the `server_default` in the next operation.